release.json

```diff
@@ -29,7 +29,7 @@
         "SECURITY_AGENT_POLICIES_VERSION": "master"
     },
     "release-a6": {
-        "INTEGRATIONS_CORE_VERSION": "7.33.0-rc.5",
+        "INTEGRATIONS_CORE_VERSION": "7.33.0-rc.6",
```

Comment:
is it expected to use the rc.6 tag of integrations-core even though we're preparing rc.7? (if it's expected, then all good 👍 )

Reply:
Between rc.6 and rc.7 in integrations-core there are no production changes, just CI fixes. I think we missed it somehow during rc6 build (due to the fact that it's not impacting any functionality) and only now this gets picked up. I think this is acceptable.